docs/general/administration/hardware-selection.md

```diff
@@ -9,55 +9,55 @@ This page covers what you need to know in order to select appropriate hardware f
+AMD is **NOT** recommended if you plan to use integrated graphics for Jellyfin.
 
-:::
+### Server with Dedicated Graphics
 
-:::tip Fully Utilizing Intel-based Macs
+If you are planning to use a dedicated graphics card (including upgrading an old system with a dedicated GPU), the following specs are recommended:
 
-It is recommended that Intel-based macs be used with Windows or Linux installed to host Jellyfin. Many hardware acceleration features aren't available on MacOS due to the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork not being available.
+- CPU: Intel Core i5-2300， AMD FX-8100 or better (Geekbench 6 Multicore 1500 or better), CPU Vendor / Performance will **NOT** affect hardware encode speed or quality
+- RAM: 8GB (4GB should be enough for a server running Linux without a GUI)
+- Graphics: Intel Arc A series or newer, Nvidia GTX16/RTX20 series or newer (Excluding GTX1650), **AMD NOT recommended**.
 
-:::
+### Low Power Servers
 
-### Low Power Applications
+For users with expensive power or running servers on battery power:
 
-For low power applications, Intel 12th gen or newer Atom CPUs with integrated graphics are recommended. It is also recommended that [Low Power Encoding](/docs/general/administration/hardware-acceleration/intel/#low-power-encoding) be setup.
+- Intel 12th gen N series platforms (Idle ~10w, Full ~15w)
+- Apple M Series mac mini (Idle ~5w, Full ~40w)
+- Rockchip RK3588 / RK3588S SBC (Idle ~5w, Full ~15w, **Advanced Users Only**)
 
-:::caution SBCs (Single Board Computers)
+### Do Not buy
 
-Most SBCs use low powered chipsets, often with less than ideal driver support from the chipset vendors. They are generally too slow for a good experience and/or have broken hardware acceleration support. Please avoid using SBCs such as Raspberry Pis (Including newer Raspberry Pi 5 Models) to run Jellyfin. **One exception** is that the **Rockchip RK3588/3588S SoC based SBCs** provide excellent transcoding speed and power efficiency - H.264/HEVC up to 1080p@480fps or 4k@120fps transcoding with **single-digit** power consumption in watts. Another bonus is that it has **10-bit H.264 (High10) and AV1 decoding** support. Note that Rockchip hardware transcoding is only supported in **Jellyfin 10.9 or above**. Since these Arm based SBCs do not use an unified installation ISO image like x86-64 based PCs do. Please check with the SBC manufacturer or community maintainer for support and documentation before purchasing.
-
-:::
+- Intel "Atom" CPUs: Intel J/M/N/Y series low power CPUs up to 11th gen use a different architecture than higher end parts, leading to subpar performance despite what their names might suggest. Please be careful about these parts when shopping for a system to run Jellyfin.
+- Prebuilt NAS devices: Prebuilt NAS devices often have low powered CPUs underpowered for a good Jellyfin experience. Please lookout for the specs
```

Comment:
The most problematic part is not the hardware, as the CPU is actually good enough for many of them, and some have already transitioned to using Alder Lake-N or similar processors. The more problematic part is the software stack required to install Jellyfin on them, which is usually very vendor-specific and may break Jellyfin, especially the hardware acceleration. If that happens, Jellyfin cannot provide much help for that specific hardware-software combination and cannot fix it.

Reply:
Many still have 11th gen or below Intel Atom or even RTL something. I will change this section to be watch out but not out right not recommend.